### PR TITLE
Instructor: edit session: add icons to buttons #7227

### LINKED ISSUE #7227: Instructor: edit session: add icons to buttons
In the page used by instructors to edit session/question details, we can add suitable icons to the buttons.

image

### PR DESCRIPTION
**Pull request by VamsiSangam**
Fixes #7227 

**Outline of Solution**

Added glyphicon span elements and updated UI test files using god mode.

### CODE REVIEW

**Comment by VamsiSangam** [comment]
Ready for review.

**Comment by damithc** [comment]
@VamsiSangam please give screenshots of the UI change

**Comment by VamsiSangam** [comment]
Sorry forgot about that :sweat_smile: . Here are the latest screenshots -

**Comment by damithc** [comment]
While we are at it, I'm wondering if we can shorten `Save Changes` to `Save` and `Discard Changes` to `Cancel`

Any other options for the `discard` icon? Sometimes `x` is used for `delete`.

**Comment by damithc** [comment]
Note that once changes to functional code is confirmed, you might have to update the instructor help page too.

**Comment by VamsiSangam** [comment]
> Note that once changes to functional code is confirmed, you might have to update the instructor help page too.

Thanks for reminding, I'll keep that in mind sir. Other alternative for `Cancel` is -



Other than that there are only [other icons](http://bootstrapdocs.com/v3.1.1/docs/components/) which are minor variations of the `x`. If we use the above icon for `Cancel` then it might make a lot of sense if we use its matching pair for `Save` which is -

**Comment by damithc** [comment]
Other options for cancel:
image
image

Other options for copy:
image

**Comment by VamsiSangam** [comment]
This is the current state of the icons -









@damithc sir, personally I think `glyphicon-triangle-left` is an inappropriate choice. `glyphicon-ban-circle` seems good. Please review the `Cancel`'s icon button currently present, I will change it to `glyphicon-ban-circle` if you feel it is more appropriate.

And sir, the version of Bootstrap we are using is 3.1.1, which does not support the `glyphicon-duplicate` icon. [Here's a list](http://bootstrapdocs.com/v3.1.1/docs/components/) of supported icons.

**Comment by VamsiSangam** [comment]
IMO `glyphicon-ban-circle` is a great choice for `Discard`, but let us wait for what @damithc sir has to say.

**Comment by damithc** [comment]
Let's try `ban` icon. Although its meaning is not a direct match, it seems to be used widely for cancel (so is `x' but that one has a chance of confusing with 'delete')

**Comment by VamsiSangam** [comment]
This is how it looks with `ban` icon. Personally, I prefer the `ban` icon too.

capture

**Comment by damithc** [comment]
Any other options for 'save'? The icon is a bit hard to recognize :-p

**Comment by VamsiSangam** [comment]
Other possible alternatives for `Save` button icon -







Here's a [list](http://bootstrapdocs.com/v3.1.1/docs/components/) of the supported icons (for reference).

**Comment by damithc** [comment]
@VamsiSangam can you post screenshots with 1st, 3rd, and 4th icons, to see how they look when used?

**Comment by VamsiSangam** [comment]
Here they are -

**Comment by damithc** [comment]
I like this one

**Comment by VamsiSangam** [comment]
Will go for the `ok` and `ban` icons. 👍 . I wanted to update icons in `instructorHelp.jsp`, but what I found was that plenty of code needs to be updated. Eg. this is the example given in `instructorHelp.jsp`, for `Sessions -> Setting up questions`.



This is the actual UI the instructor sees when creating a new essay question -



And not just that but, example UI for adding a new question, Multiple-choice (single answer) question are also outdated. And I suspect a lot more examples are outdated.

Shall I create another issue which concerns itself only with updating the UI of examples?

**Comment by damithc** [comment]
> Shall I create another issue which concerns itself only with updating the UI of examples?

Yes, can be a separate issue in that case.

File: src/test/resources/pages/instructorStudentListPageSearchStudentMultiple.html [hunk 1]
```diff
@@ -135,133 +135,6 @@
                   .tmms@gmail.tmt
                 </td>
                 <td class="no-print align-center">
-                  <a class="btn btn-default btn-xs" data-original-title="View the details of the student" data-placement="top" data-toggle="tooltip" href="/page/instructorCourseStudentDetailsPage?courseid=course2&studentemail=CCSDetailsUiT.alice.tmms%40gmail.tmt&user=instructorWith2Courses" rel="noopener noreferrer" target="_blank" title="">
```
Comment (VamsiSangam) [inline src/test/resources/pages/instructorStudentListPageSearchStudentMultiple.html]: Unintentional changes occurred. Will make necessary modifications.

**Comment by VamsiSangam** [comment]
Ready for review.

**Pull request merged**